### DISCUSSION TITLE
hard coded foreigner account ID appears in the IAM policy! scratching my head, please help

### DISCUSSION THREAD

**Discussion by ManarAbd** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#562] [General]
The stack is failing for me when it tries to create the dashboard, with the below error. has anyone seen this issue?
Why does it tries to get to a hard coded AWS account ID = default value for cidAccountId in cid/helpers/quicksight/init.py ??
I am authenticating to AWS using single sign on, QuickSight and s3 bucket have been provisioned in ap-southeast-2.

Error: waiting for CloudFormation Stack (arn:aws:cloudformation:ap-southeast-2:MYACCOUNTID:stack/Cloud-Intelligence-Dashboards/xx-1693-11ee-xx-xxx) create: failed to create CloudFormation stack, rollback requested (ROLLBACK_COMPLETE): ["The following resource(s) failed to create: [CostIntelligenceDashboard]. Rollback requested by user." "Received response status [FAILED] from custom resource. Message returned: Error: An error occurred (AccessDeniedException) when calling the DescribeTemplate operation: User: arn:aws:sts::MYACCOUNTID:assumed-role/CidExecRole/CidCustomResourceDashboard is not authorized to perform: quicksight:DescribeTemplate on resource: arn:aws:quicksight:us-east-1:223485597511:template/Cost_Intelligence_Dashboard with an explicit deny in a service control policy - Cannot find Cost_Intelligence_Dashboard in account 223485597511.

**Comment by iakov-gan** [comment]
Hello

Dashboards are deployed from templates hosted in our account. Templates are un us-east-1 and cannot be in other regions. 

Dashboards can be deployed in any region but you need to allow describe template * in us-east-1 region in your scp.
